### DISCUSSION TITLE
Preventivi, inserimento anagrafica cliente, APIv1 Vs. APIv2

### DISCUSSION THREAD

**Discussion by mastsolutions** [fattureincloud/api#199] [Q&A] [answered]
Buongiorno,
con l'inserimento di un preventivo, con API v1 passavo solo il codice cliente + Nome e veniva recuperata/inserita l'intera anagrafica (Via, P.IVA, ecc.). Con API v2 passando gli stessi dati (cod. cliente e nome) viene visualizzato solo il nome cliente la mail e il telefono. Con APIv2 devo passare tutti i dati perchè siano visualizzati nel preventivo?
Grazie
Stefano

**Comment by emajo** [comment, marked as answer]
Ciao @mastsolutions, esatto con le API v2 attualmente è necessario passare tutti i dati anche specificando il codice del cliente, abbiamo intenzione di replicare la funzionalità presente nelle API v1 ma non abbiamo ancora una tempistica per questa modifica.

Grazie,
Emanuele.

**Comment by mastsolutions** [comment]
Grazie Emanuele
